src/components/Executions/ExecutionDetails/ExecutionNodeViews.tsx

```diff
@@ -85,29 +86,33 @@ export const ExecutionNodeViews: React.FC<ExecutionNodeViewsProps> = ({
                 <Tab value={tabs.nodes.id} label={tabs.nodes.label} />
                 <Tab value={tabs.graph.id} label={tabs.graph.label} />
             </Tabs>
-            <div className={styles.nodesContainer}>
-                {tabState.value === tabs.nodes.id && (
-                    <>
-                        <div className={styles.filters}>
-                            <ExecutionFilters {...filterState} />
-                        </div>
+            <NodeExecutionDetailsContextProvider
```

Comment:
The only change here: wrapped Tabs content with NodeExecutionDetailsContextProvider. github compare just gone rogue :)